RFCs/[RFC-003-RTVF]-RTA-Transaction-Validation-Flow.md

```diff
@@ -109,38 +109,45 @@ To validate RTA Transaction graftnode should perform several checks:
    
       > **TODO:** _Boris:_ I'm leaning towards trusting PoS proxy with respect to auth sample but use the PoS one-time identification keypair to enforce anonymity.
    
->**Payment Block Definition**
->
->  **Payment block** is a historical block in the blockchain, which selected by the block number as a difference between current blockchain height and constant value, which determines the delay for increasing the stability of selected auth sample (Currently we use SVP). Formally,`payment_block_number = current_block_number - SVP`. Block defined by using its block number and block hash.
+    >**Payment Block Definition**
+    >
+    >  **Payment block** is a historical block in the blockchain, which selected by the block number as a difference between current blockchain height and constant value, which determines the delay for increasing the stability of selected auth sample (Currently we use SVP). Formally,`payment_block_number = current_block_number - SVP`. Block defined by using its block number and block hash.
+
+    > **Purchase details**  is an information about purchase - may include list of pushase items, price and amount of each item, etc
+
+    > **Payment data** is a data structure containing **encrypted purchase details** and total amount  (**FullTxAmount** in atomic units)
+
+2. When PoS got data from Proxy Supernode, it prepares and sends **encrypted payment data** (to `/dapi/sale` endpoint):
+    * ~~generates the symmetric encryption key, called **PoS data encryption key**. This key will be only passed to Wallet app (only Wallet app can read this payment details)~~
```

Comment:
Is the symmetric key to be passed through the QR code? To be one the safe side, why don't use Deffi-Helman over the net instead?

Reply:
1. as I mentioned right now we don't have interfaces to encrypt/decrypt using symmetric key and **error checking** (guess it's not hard to implement, but still), so I suggested to use interfaces what we already have (multiple recipients encryption, just use one key for a wallet)  and pass private key via QR code.
2. POS and Wallet isn't connected over network directly, in case we decide to use DH or some other protocol - this would require to change design/API